### PR TITLE
Arena Beginner Set - adding to provide a home for Shorecomber Crab

### PR DESCRIPTION
**Pull request by tiera3**
Arena Beginner Set

Shorecomber Crab is currently listed in ArenaNewPlayerExperienceExtras but doesn't belong there. On scryfall, the only set it exists in is ANB which is not currently on xmage, so I added a file for that set.

### CODE REVIEW

File: Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSupportCards.java [hunk 1]
```diff
@@ -543,6 +543,7 @@ public class ScryfallImageSupportCards {
             add("MKM"); // Murders at Karlov Manor
             add("MKC"); // Murders at Karlov Manor Commander
             add("CLU"); // Ravnica: Clue Edition
+            add("ANB"); // Arena Beginner Set
```
Comment (xenohedron) [inline Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSupportCards.java:546]: need to fix merge conflict, better to put this in chronological order (scryfall lists a date of 2020-08-13)

File: Mage.Sets/src/mage/sets/ArenaBeginnerSet.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        cards.add(new SetCardInfo("Air Elemental", 23, Rarity.UNCOMMON, mage.cards.a.AirElemental.class));
+        cards.add(new SetCardInfo("Angelic Guardian", 2, Rarity.RARE, mage.cards.a.AngelicGuardian.class));
+        cards.add(new SetCardInfo("Angelic Reward", 3, Rarity.UNCOMMON, mage.cards.a.AngelicReward.class));
+        cards.add(new SetCardInfo("Angel of Vitality", 1, Rarity.UNCOMMON, mage.cards.a.AngelOfVitality.class));
+        cards.add(new SetCardInfo("Arcane Signet", 117, Rarity.COMMON, mage.cards.a.ArcaneSignet.class));
+        cards.add(new SetCardInfo("Armored Whirl Turtle", 24, Rarity.COMMON, mage.cards.a.ArmoredWhirlTurtle.class));
+        cards.add(new SetCardInfo("Bad Deal", 45, Rarity.UNCOMMON, mage.cards.b.BadDeal.class));
+//        cards.add(new SetCardInfo("Baloth Packhunter", 90, Rarity.COMMON, mage.cards.b.BalothPackhunter.class));
+        cards.add(new SetCardInfo("Bombard", 67, Rarity.COMMON, mage.cards.b.Bombard.class));
+//        cards.add(new SetCardInfo("Bond of Discipline", 4, Rarity.UNCOMMON, mage.cards.b.BondofDiscipline.class));
```
Comment (xenohedron) [inline Mage.Sets/src/mage/sets/ArenaBeginnerSet.java:36]: the issue is you need to capitalize "Of" in the class name

File: Mage.Sets/src/mage/sets/ArenaBeginnerSet.java [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+    public static ArenaBeginnerSet getInstance() {
+        return instance;
+    }
+
+//  Missing cards - Baloth Packhunter , Compound Fracture , Hallowed Priest , Mardu Outrider , Tin Street Cadet
+//  Removed card - Bond of Discipline because it caused a symbol not found error
+
+    private ArenaBeginnerSet() {
+        super("Arena Beginner Set", "ANB", ExpansionSet.buildDate(2024, 2, 24), SetType.MAGIC_ONLINE);
```
Comment (xenohedron) [inline Mage.Sets/src/mage/sets/ArenaBeginnerSet.java:22]: see other comment re: date

**Review by xenohedron — COMMENTED** [review]
miss remove Shorecomber Crab from the set where it is incorrectly listed?

why is the _entire_ mtg-sets-data file modified? that can't be right

please rebase against latest master

**Comment by tiera3** [comment]
> miss remove Shorecomber Crab from the set where it is incorrectly listed?
Re- removing Shorecomber Crab from XANA. I was planning on doing that as a followup - IF this set was accepted. (I had no idea if it had been deliberately excluded for some reason.)
 
> why is the _entire_ mtg-sets-data file modified? that can't be right
It was too large to edit online, so I had to download it, append lines, then re-upload. I don't know why it counted it as the whole file being modified. (I didn't check the EOL character. That is a possibility.)

> please rebase against latest master
Will attempt to do so.

**Comment by JayDi85** [comment]
Make sure git uses autocrlf - it will auto convert line endings:
https://github.com/magefree/mage/wiki/Setting-up-your-Development-Environment#ide

**Pull request closed without merging**